echo_handlers.go

```diff
@@ -238,6 +236,10 @@ func (svc *Service) AppsNewHandler(c echo.Context) error {
 	}
 	disabled := c.QueryParam("editable") == "false"
 	requestMethods := c.QueryParam("request_methods")
+	if requestMethods == "" {
```

Comment:
could the readme be updated on how to pass one or more request_methods in the query?

Reply:
http://localhost:8080/apps/new?request_methods=pay_invoice%20request_methods=read_balance does not seem to work